### ISSUE TITLE
Authentication doesn't stick

### ISSUE THREAD

**Issue by philkernick** [joshhunt/destinySets#287]
Every time I go to destinysets it asks me to login.  After I login it works fine.  Until I close the browser.

Whatever cookie or authentication method you are using isn't working.

**Comment by joshhunt** [comment]
Yeah, this is unfortunately expected behaviour due to the recommended
authentication method that Destiny Sets uses, and "enhancements" to browser
security and privacy that prevent it from refreshing authentication an hour
after you log in.
